pom.xml

```diff
@@ -147,10 +146,47 @@
 			<version>1.6.2</version>
 		</dependency>
 		<dependency>
-			<groupId>com.xceptance</groupId>
-			<artifactId>neodymium-library</artifactId>
-			<version>${neodymium.version}</version>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
+			<version>4.13</version>
 		</dependency>
+		<dependency>
+			<groupId>io.qameta.allure</groupId>
+			<artifactId>allure-junit4</artifactId>
```

Comment:
IMO, we don't need Allure in this project. I don't think its even really used. So we should remove it with all dependencies.